src/jxl.imageio/jxlinput.cpp

```diff
@@ -320,29 +344,17 @@ JxlInput::read_native_scanline(int subimage, int miplevel, int y, int /*z*/,
     lock_guard lock(*this);
     if (!seek_subimage(subimage, miplevel))
         return false;
-    if (y < 0 /* || y >= (int)m_cinfo.output_height*/)  // out of range scanline
+    if (y < 0)  // out of range scanline
```

Comment:
Should check for y too big here, too.

Reply:
Is that correct now? Or shoud be greater or equal to height?

Reply:
`>=`.  If height is 10, the scanlines range from 0 to 9, so 10 would result in the subsequent memcpy misbehaving.